css/mobile.css

```diff
@@ -21,22 +21,18 @@
     .logo {
       display: block;
       position: relative;
-      width: 100%;
-      height: 30%;
-      background-image: url(../assets/Cortlandt-Labs-logo.png);
       background-size: contain;
       background-repeat: no-repeat;
       background-position: center;
+      margin: 0 auto;
     }
-    .mission-statement.mobile {
-      display: block;
+    .mission-statement {
       position: relative;
       font-size: 4.5em;
+      line-height: 1em;
+      padding-top: 33vh;
```

Comment:
To fit all components in firstSegment in mobile, I think under width 450px, mission statement needs padding-top 18vh